orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -236,21 +235,20 @@ private static void outputReport(String species) throws IOException
 		// Create file if it doesn't exist
 		String reportFilename = "report_ortho_inference_test_reactome_" + releaseVersion + ".txt";
 		logger.info("Updating " + reportFilename);
-		File reportFile = new File(reportFilename);
-		reportFile.createNewFile();
+		createNewFile(reportFilename);
 		String results = "hsap to " + species + ":\t" + inferredCount + " out of " + eligibleCount + " eligible reactions (" + String.format("%.2f", percentInferred) + "%)\n";
 		Files.write(Paths.get(reportFilename), results.getBytes(), StandardOpenOption.APPEND);
 	}
 	
 	// Statically store the adaptor variable in each class
-	private static void setDbAdaptors(MySQLAdaptor dbAdaptor2) 
+	private static void setDbAdaptors()
```

Comment:
Adding a check in this method to ensure the dbAdaptor being used isn't null when the method is called would be great.

Reply:
If the database can't be connected to the MySQLAdaptor class throws a `MySQLNonTransientConnectionException`, so I think adding additional guards would be redundant

Reply:
Sure, that's fair.  It may be a good, though, to put a check in just in case the method is ever called in some way a null does make it in.  A little bit of paranoid redundancy here could be helpful before setting all adaptors and so this code is protected if there are ever changes to MySQLAdaptor class.